experimental/SetPartitions/src/Util.jl

```diff
@@ -0,0 +1,95 @@
+"""
+    helper_new_point_values_vector(p::Vector{Vector{Int}}, q::Vector{Vector{Int}})
+
+Return a semantically identical partition in form of an vector, 
+which has new number values.
+"""
+function helper_new_point_values_vector(p::Vector, q::Vector)
+
+    p_points = vcat(copy(p[1]), copy(p[2]))
+
+    if !(isempty(p_points))
+        new_id, index = findmax(p_points)
+        new_id += 1
+    else
+        new_id = 1 
+    end
+    
+    q_points = vcat(copy(q[1]), copy(q[2]))
+    new_ids = Dict()
+
+    for n in q_points
+        new_ids[n] = new_id
+        new_id += 1
+    end
+    upper = []
+    for (i, n) in enumerate(q[1])
+        push!(upper, get(new_ids, n, -1))
+    end
+    
+    lower = []
+    for (i, n) in enumerate(q[2])
+        push!(lower, get(new_ids, n, -1))
+    end
+    
+    [upper, lower]
+    
+end
+
+"""
+    normal_form_vector(p::Vector)
+
+Return a semantically identical partition of vector form, 
+which has new number values from 1 to number of blocks of `p`.
+"""
+function normal_form_vector(p::Vector)
+
+    new_id = 1
+    new_ids = Dict{Int, Int}()
+    p_return = helper_new_point_values_vector([[length(p[1]) + length(p[2])], []], p)
+
+    for (i, n) in enumerate(p_return[1])
+        if !(n in keys(new_ids))
+            new_ids[n] = new_id
+            p_return[1][i] = new_id
+            new_id += 1
+        else
+            p_return[1][i] = get(new_ids, p_return[1][i], -1)
```

Comment:
Huh, isn't this the sameas
```suggestion
            p_return[1][i] = get(new_ids, n, -1)
```